pkg/client/src/app/pages/applications/analysis-wizard/analysis-wizard.tsx

```diff
@@ -110,18 +112,7 @@ export const AnalysisWizard: React.FunctionComponent<IAnalysisWizard> = ({
     },
   });
 
-  const { identities, fetchIdentities } = useFetchIdentities();
-
-  React.useEffect(() => {
-    fetchIdentities();
-  }, [fetchIdentities]);
-
-  const hasIdentity = (application: Application, kind: string): boolean =>
-    !!application.identities?.some((appIdentity) =>
-      identities?.find(
-        (identity) => appIdentity.id === identity.id && identity.kind === kind
-      )
-    );
+  React.useEffect(() => fetchIdentities(), [fetchIdentities]);
```

Comment:
What do we gain from having a use effect here? One of the perks of using react-query is that it eliminates the need for useEffect. This might be a bit of an anti-pattern. If we want to have a reusable function for filtering identities, I would suggest adding a utility function rather than a hook here.